include/picongpu/fields/incidentField/profiles/GaussianPulse.hpp

```diff
@@ -263,10 +263,8 @@ namespace picongpu::fields::incidentField
                                 - pmacc::math::Pi<float_X>::doubleValue / Unitless::WAVE_LENGTH * r
                                 + (2._X * float_X(m) + 1._X) * xi + phase + typename Unitless::LAGUERREPHASES_t{}[m]);
                     }
+                    auto const shiftedTime = time - r / SPEED_OF_LIGHT;

```

Comment:
this changes the physics
Could you please elaborate why this is still true?

Reply:
Well, the time now runs in positive direction, this doesn't change anything for the symmetric Gaussian. And phase offset does not shift the envelope anymore, but I don't think this was ever desired behavior?